snmp/tests/test_profiles.py

```diff
@@ -1266,7 +1267,7 @@ def test_aruba(aggregator):
     aggregator.assert_all_metrics_covered()
 
 
-def test_chatsworth(aggregator):
+def test_chatsworth_new(aggregator):
```

Comment:
Renamed to make it easier to filter.

Previously `ddev test snmp:py38 -k test_chatsworth` will also match `test_chatsworth_legacy`.